homeassistant/components/totalconnect/alarm_control_panel.py

```diff
@@ -59,9 +59,15 @@ def state(self):
         """Return the state of the device."""
         return self._state
 
+    @property
+    def state_attributes(self):
```

Comment:
You are not allowed to implement state_attributes. Implement `device_state_attributes` instead.

Reply:
Updated to `device_state_attributes`